client/src/components/Form/components/formSections/ArbeidsevneSection.tsx

```diff
@@ -30,6 +30,7 @@ const ArbeidsevneSection = ({ section, setSchema, schema, errors, validate }: Ar
         <SectionContainer section={section}>
             <Subsection sectionIdentifier="7.1">
                 <Checkbox
+                    disabled
```

Comment:
Tror ikke vi bør disable disse. Bedre at vi bare ignorerer det som er skrevet inn i feltene ved innsending, ellers kan det virke forvirrende ved test

Reply:
blir mye merarbeid for den som fyller inn isåfall. hvis de er disabled så er det lett å se hva som valideres

Reply:
Forsåvidt enig. Det jeg tenkte er at hvis disse er disabled, så vil vi få spørsmål om hvorfor de er disabled.

Kan la det være i denne omgang.